### PR TITLE
fix: add heading styles

### PR DESCRIPTION
**Pull request by kodisu**
Tailwind defaults to removing styles for headings (and other things like lists) to follow modern normalize. https://tailwindcss.com/docs/preflight

I believe the convention is to keep preflight on and to overwrite with your own styling.

So we should decide what our base styles are and also what our theme for colors, fonts, spacing etc should be, according to this doc: https://tailwindcss.com/docs/adding-custom-styles. For now we should definitely have the headings be different size text font

COLORS:
DaisyUI sits on top of tailwind, so we will follow their semantic color first. Any custom colors (like blue-light and blue-dark) will from tailwind. https://daisyui.com/docs/colors/ https://tailwindcss.com/docs/theme

TLDR: follow the colors on the daisy-spike.svelte page

### CODE REVIEW

**Review by anna-giang — APPROVED** [review]
LGTM! Thanks Cody!
